src/BODY/pair_body_rounded_polygon.cpp

```diff
@@ -590,19 +588,17 @@ void PairBodyRoundedPolygon::body2space(int i)
 
 void PairBodyRoundedPolygon::sphere_against_sphere(int i, int j,
                        double delx, double dely, double delz, double rsq,
-                       double k_n, double k_na, double** /*x*/, double** v,
+                       double k_n, double k_na, double** /*x*/, double** /*v*/,
                        double** f, int evflag)
 {
   double rradi,rradj;
-  double vr1,vr2,vr3,vnnr,vn1,vn2,vn3,vt1,vt2,vt3;
-  double rij,rsqinv,R,fx,fy,fz,fpair,shift,energy;
+  double rij,R,fx,fy,fz,fpair,shift,energy;
```

Comment:
@ndtrung81 please have a look at this change. the removed code had no effect, but it is not clear if this is just removing dead code, or maybe there is a bug, that the results of the removed code where not applied and thus this should be reverted and the missing statement(s) added.

Reply:
@akohlmey I believe this is a bug in my part, and there are missing variables and statements that should be added:

    double vr1,vr2,vr3,vnnr,vn1,vn2,vn3,vt1,vt2,vt3;
    double rij,rsqinv,R,fx,fy,fz,fn[3],ft[3],fpair,shift,energy;
    ...
    if (R <= EPSILON) { // in contact
       ...
       vt3 = vr3 - vn3;

       // normal friction term at contact

       fn[0] = -c_n * vn1;
       fn[1] = -c_n * vn2;
       fn[2] = -c_n * vn3;

       // tangential friction term at contact,
       // excluding the tangential deformation term for now

       ft[0] = -c_t * vt1;
       ft[1] = -c_t * vt2;
       ft[2] = -c_t * vt3;

       fx += fn[0] + ft[0];
       fy += fn[1] + ft[1];
       fz += fn[2] + ft[2];
     }

Attached is the diff.

[diff_pair_body_rounded_polygon.txt](https://github.com/lammps/lammps/files/4855843/diff_pair_body_rounded_polygon.txt)